clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -850,12 +850,12 @@ void CodeGenModule::Release() {
     // Indicate whether __nvvm_reflect should be configured to flush denormal
     // floating point values to 0.  (This corresponds to its "__CUDA_FTZ"
     // property.)
-    getModule().addModuleFlag(llvm::Module::Override, "nvvm-reflect-ftz",
+    getModule().addModuleFlag(llvm::Module::Max, "nvvm-reflect-ftz",
```

Comment:
Can you please explain what Max means here and Override meant before?

Reply:
See here https://github.com/intel/llvm/blob/sycl/llvm/include/llvm/IR/Module.h#L136 . Here Max means that the max value will become the global value at `llvm-link` time (which corresponds to FTZ=true). The `Override` option would instead give an error at llvm-link time for multiple definitions of the same module flag

Reply:
Thank you!